agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -84,17 +84,22 @@ Sponsor will only do so if Assignor does not sign the documents within a reasona
 
 ### Applicable Asset Categories
 
-This agreement covers the transfer of assets in the following categories.
-The specific assets being transferred are detailed in the corresponding schedules:
+This Agreement may be used to transfer ownership of Project assets in the categories listed below.
+Transfer of a specific asset occurs only when the corresponding schedule is completed and incorporated into an Asset Transfer Agreement executed by the parties.
```

Comment:
Should "the parties" be "both parties" or "all parties"?

Reply:
"The parties" is clear enough, as the agreement can't be executed w/o them all. This is mostly re-affirming that there should be a signed agreement to go alongside 1..n schedules.